### ISSUE TITLE
YM2612 emulation core might want to be updated.

### ISSUE THREAD

**Issue by brickblock369** [kuma4649/MDPlayer#247]
YM2612 emulation core needs to be updated, as Comix Zone's "Episode 1, Page 2-2" still plays incorrect frequencies, and SSG-EG speed doesn't seem to match, as evidenced from the [VGM file of this song.](https://www.youtube.com/watch?v=IV__UlazPd4)

**Comment by kuma4649** [comment]
エミュレーションの種類は何でしょうか？
問題がでるのはMDPlayerのみですか、それとも元のエミュレーションも問題が発生しますか？

**Comment by brickblock369** [comment]
I assume you're using VGMPlay for Mega Drive playback.
I've tried playing it on VGMPlay's recent version, and it plays just fine.

**Comment by kuma4649** [comment]
VGMPlayはOPN2のコアを複数種持っていたかと思うのですが、どれを使用していますか。
また、MDPlayerのほうではコアは何を選んでいますか。

**Comment by kuma4649** [comment]
こちらどうなりましたか

**Comment by brickblock369** [comment]
Sorry for the late reply, I didn't see your comment until now
.
I'm using MAME for VGMPlay. Both songs sounded like it should.

I did try Nuked-OPN2 in VGMPlay, and while Blue Electric sounded like it should, Comix Zone has a completely different problem, though it also might be a fault inside the VGM itself.

MDPlayer seems to be using Gens emulator? I hear that exact same frequency issue in Comix Zone when I set my VGMPlay to Gens.
